arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1645,6 +1645,11 @@
 			</dict>
 			<dict>
 				<key>descriptionText</key>
+				<string>Create graphics for utility associations in a utility network.</string>
+				<key>storyboardName</key>
+				<string>DisplayUtilityAssociations</string>
+				<key>displayName</key>
+				<string>Display utility associations</string>
```

Comment:
This isn't right. The key-value pairs for this sample should be in their own dictionary.

![image](https://user-images.githubusercontent.com/2257493/82367109-20249a00-99c8-11ea-80ee-c21033421ffc.png)

Please fix so that I can build and finish re-reviewing this PR.

Reply:
@philium It has been fixed. I also reindented the file in case my editing threw if off somehow.